src/ticketmap/index.tsx

```diff
@@ -61,6 +69,28 @@ interface TicketGroupsBySectionByZone {
   }
 }
 
+const createDeepEqualSelector = createSelectorCreator(defaultMemoize, isEqual)
+
+const $missingSectionIds = createDeepEqualSelector(
+  (state: State) => state.ticketGroups,
+  (state: State) => state.sectionZoneMapping,
+  (ticketGroups, sectionZoneMapping) => ticketGroups
+    .map(ticketGroup => ticketGroup.tevo_section_name.toLowerCase())
+    .filter(sectionId => sectionZoneMapping[sectionId] === undefined)
+)
+
+const $availableTicketGroups = createDeepEqualSelector(
+  (state: State) => state.ticketGroups,
+  (state: State) => state.sectionZoneMapping,
+  (ticketGroups, sectionZoneMapping) => ticketGroups
+    .map(ticketGroup => ({
+      section: ticketGroup.tevo_section_name,
+      zone: sectionZoneMapping[ticketGroup.tevo_section_name.toLowerCase()],
+      price: ticketGroup.retail_price
+    } as NormalizedTicketGroup))
+    .filter(ticketGroup => ticketGroup.zone !== undefined)
```

Comment:
We want to include ticket groups without zones, at this point I believe zones are optional and many ticket groups (if not all) don't have zone information.

Reply:
We also need to filter out ticket groups which don't match a section in the manifest.

Reply:
`TicketMap.getAvailableTicketGroups` used `sectionZoneMapping` as a way to test whether a ticketGroup is available. I just moved that logic into a selector. We could update the logic in ano ther branch.

Reply:
Correct, currently in `getAvailableTicketGroups` if `this.state.sectionZoneMapping[section]` is falsey, that ticket group is not added to the list of available tickets. I don't see that logic in the selector.